README.md

```diff
@@ -12,8 +12,9 @@ $ npm install --save oniyi-http-plugin-format-url-template
 const OniyiHttpClient = require('oniyi-http-client');
 const oniyiHttpPluginUrlTemplate = require('oniyi-http-plugin-format-url-template');
 
-const clientOptions = {};
-const httpClient = new OniyiHttpClient(clientOptions);
+const options = {
+  requestPhases: ['format-url-template'], // indicates that we want phase hook handler with name 'format-url-template' should be run in request phase list
```

Comment:
request phases MUST include the phase names `initial` and `final`, since `oniyi-http-client` uses [phaseList.zipMerge()](https://github.com/strongloop/loopback-phase/blob/master/lib/phase-list.js#L164) to merge whatever is provided here with the default phase names `['initial', 'final']`.
Without providing these two, you can not control in what order your custom phase is added

Reply:
agree